_docs/bug-bounty.md

```diff
@@ -44,11 +44,9 @@ The OAK team is eager to engage developers in building the OAK Network ecosystem
 4. Once your submission is accepted, we will contact you to collect your wallet information to issue the rewards.
 
 ## Scope
-Currently, the following OAK Network code repositories are in scope for bug bounty program:<br>
-<https://github.com/OAK-Foundation/OAK-blockchain/tree/oak-testnet/node/runtime><br>
-<https://github.com/OAK-Foundation/OAK-blockchain/tree/oak-testnet/pallets/quadratic-funding><br>
-<https://github.com/OAK-Foundation/OAK-blockchain/tree/oak-testnet/node/cli><br>
-<https://github.com/OAK-Foundation/quadratic-funding-webapp><br>
+Currently, the following OAK Network code repositories are in scope for bug bounty program:
+- [OAK's blockchain code](https://github.com/OAK-Foundation/OAK-blockchain)
+- [Parachain Staking Module](https://github.com/OAK-Foundation/moonbeam/tree/master/pallets/parachain-staking)
```

Comment:
let's use `https://github.com/OAK-Foundation/moonbeam/tree/oak-staging/pallets/parachain-staking` instead so that the modifications that we've made are in-scope.

Reply:
whew good call